include/odp/api/spec/packet.h

```diff
@@ -244,6 +267,16 @@ odp_packet_t odp_packet_from_event(odp_event_t ev);
  */
 odp_event_t odp_packet_to_event(odp_packet_t pkt);
 
+/**
+ * Convert multiple packet handles to events
+ *
+ * @param      pkt  Array of packet handles to convert
+ * @param[out] ev   Event handle array for output
+ * @param      num  Number of packets and events
+ */
+void odp_packet_to_event_multi(const odp_packet_t pkt[], odp_event_t ev[],
+			       int num);
```

Comment:
Again since `odp_packet_to_event()` is likely to be just a cast that can be inlined away it's not clear why a `multi` version is needed.

Reply:
The same answer. Improved implementation efficiency and cleaner application code.